unit_tests/EnumUnitTests.d

```diff
@@ -248,6 +248,64 @@ D", options);
 
 }
 
+// Test aliasing of all enum members in nested scope
+unittest
+{
+    Options options;
+    options.aliasEnumMembers = true;
+
+    assertTranslates(
+q"C
+enum Enum
+{
+    FOO,
+};
+
+struct Foo
+{
+    enum FooEnum
+    {
+        BAR,
+        BAZ,
+    };
```

Comment:
No, structs without fields are essentially undefined behavior in C. They are allowed in C++ and other OOP languages, because they're useful for defining interfaces and abstract base classes, but since there's no such​ concepts in C, they're useless and hence undefined by the C standard, IIRC.